cfme/tests/infrastructure/test_scvmm_specific.py

```diff
@@ -6,6 +6,12 @@
 from cfme.infrastructure.provider.scvmm import SCVMMProvider
 
 
+pytestmark = [
+    pytest.mark.provider([SCVMMProvider], scope="module"),
+    pytest.mark.usefixtures("setup_provider_modscope")
+]
+
```

Comment:
Question:
- We have already provided ```scope="module"``` for ```SCVMMProvider``` then why do we need ```setup_provider_modscope```? Cant we use ```setup_provider```?

Reply:
As far as I know, I think it is still necessary to use `setup_provider_modscope`. https://github.com/ManageIQ/integration_tests/blob/d6be60884001d80ddf5e5320beb5ad7f5cf7d0fc/cfme/fixtures/provider.py#L284